src/test/java/org/minions/devfund/richard/moviedatabase/MovieDatabaseTest.java

```diff
@@ -0,0 +1,109 @@
+package org.minions.devfund.richard.moviedatabase;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test Class.
+ */
+public class MovieDatabaseTest {
+
+    private MovieDatabase movieDatabase;
+    private static final String MOVIE_ONE = "Movie1";
+    private static final String MOVIE_TWO = "Movie2";
+    private static final String ACTOR_ONE = "Actor1";
+    private static final String ACTOR_TWO = "Actor2";
+    private static final double TEN = 10.0;
+    private static final double TWENTY = 20.0;
```

Comment:
Don't create constants.
Please create final local variables inside test method.

Reply:
If I do that, IntelliJ tell that we should create a global variables.
![image](https://user-images.githubusercontent.com/28739313/44294723-18bbe400-a26a-11e8-82ed-35058c05623c.png)